apmpackage/apm/data_stream/sampled_traces/manifest.yml

```diff
@@ -10,11 +10,14 @@ elasticsearch:
     # This data stream holds only non-sensitive information.
     indices: [auto_configure, create_doc, maintenance, monitor, read]
   index_template:
-    settings:
-      # Create a single shard per index, so we can use
-      # global checkpoints as a way of limiting search
-      # results.
-      number_of_shards: 1
+    # settings:
+    # Create a single shard per index, so we can use
+    # global checkpoints as a way of limiting search
+    # results.
+
+    # We are not setting this for now until the package-spec is updated
+    # to allow number_of_shards: https://github.com/elastic/package-spec/issues/644
+    # number_of_shards: 1
```

Comment:
This should be already allowed :slightly_smiling_face:

Reply:
Thanks @jsoriano!

Also, I'm getting this error when building the package:
> package with GA version (8.12.0) is using an unreleased version of the spec (3.0.1-next) (PSR00001)

To reflect the change allowing `index: false` in mapping in manifests.yml file, we need 3.0.1 spec, but seems like it is not released.

Reply:
Ah yes, sorry, you would need to update to elastic-package 0.92, that includes Package Spec 3.0.1.